src/components/form/form.tsx

```diff
@@ -96,6 +103,8 @@ export class Form {
 
     public constructor() {
         this.handleChange = this.handleChange.bind(this);
+        this.getCustomErrorMessages = this.getCustomErrorMessages.bind(this);
+        this.transformErrors = this.transformErrors.bind(this);
```

Comment:
I don't think we should bind this function to `this`. `this.transformErrors` is an external function, and binding it to `this` will give it access to the internals of the form component. Although, I guess, since it's called as a method of `this` it might already have that access… 🤔

Regardless, I don't think we need this bind, and if we don't we should remove it.

Reply:
Hmm, I'll have a look at this!

Reply:
I had to look into whether internals are actually exposed, and indeed they are. I have a fix, that I can push if ok with you?

Reply:
Yes of course, please do!

Reply:
⚡

Reply:
Fixed. I'll squash and approve, if you're happy?

Reply:
Great, looks good to me!

Reply:
I tried to come up with a different prop name for `limel-form` than `transformErrors` to differentiate from rjsf, but failed. It feels clear concise to me, but feel free to change it.

Reply:
Nah, I think `transformErrors` is great. No need to _differentiate_ from rjsf, just to _decouple_ 👍